bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Widget.java

```diff
@@ -1232,6 +1232,7 @@ boolean sendMouseEvent (int type, int button, long hwnd, long lParam) {
 boolean sendMouseEvent (int type, int button, int count, int detail, boolean send, long hwnd, long lParam) {
 	if (!hooks (type) && !filters (type)) return true;
 	Event event = new Event ();
+	event.widget = this;
```

Comment:
This change seems to be unrelated to this PR. Is this some independent bug? Can we extract that change?

Reply:
It was not needed anymore. Removed it.